StRoot/StMuDSTMaker/COMMON/StMuFwdTrack.cxx

```diff
@@ -19,6 +19,9 @@ void StMuFwdTrack::set( StFwdTrack * evTrack) {
     mCharge = evTrack->charge();
     mPrimaryMomentum = TVector3( evTrack->momentum().x(), evTrack->momentum().y(), evTrack->momentum().z() );
 
+    mIdTruth = evTrack->idTruth();
+    mQATruth = evTrack->qaTruth();
+    
```

Comment:
From what I can tell, you rely on the default behavior of C++ to initialize the value of StEvent/StFwdTrack.cxx::mIdTruth to zero (for all real tracks), as it isn't explicitly initialized to have a default value neither there nor here. Is this the desired approach?